Tests/XCUITests/DatabaseFixtureTest.swift

```diff
@@ -6,6 +6,7 @@ import XCTest
 
 class DatabaseFixtureTest: BaseTestCase {
     let fixtures = ["testBookmarksDatabaseFixture": "testBookmarksDatabase1000-places.db", "testHistoryDatabaseFixture": "testHistoryDatabase100-places.db"]
+    let TIMEOUT: TimeInterval = 15
 
     override func setUp() {
```

Comment:
Do you want to disable animations for this test suite too? or it is fine as it is?